docs/contracts/deploying-a-contract.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Deploying a Contract"
+---
+
+Here we will show you how you can deploy and interact with a smart contract on Substrate.
+
+## Starting Your Node
+
+After successfully [installing Substrate](quickstart/installing-substrate.md), you can start a local development chain by running:
+
+```bash
+substrate --dev
+```
+
+![An image of the terminal starting a Substrate node](./images/start-substrate-node.png)
+
+> **Note:** If you have run this command in the past, you probably want to purge your chain so that you run through this tutorial with a clean slate. You can do this easily with `substrate purge-chain --dev`.
+
+You should start to see blocks being produced by your node in your terminal.
+
+You can interact with your node using the Polkadot UI:
+
+https://polkadot.js.org/apps/
+
+> **Note:** You will need to use Google Chrome to have this site interact with your local node. The Polkadot UI is hosted on a secure server, and your local node is not, which may cause compatibility issues on Firefox or Linux based Chromium. The other option is to [clone and run the Polkadot UI locally](https://github.com/polkadot-js/apps).
+
+If you go into the **Explorer** tab of the UI, you should also see blocks being produced!
```

Comment:
We need to set the node to be local before interacting with other parts.

Reply:
What do you mean by this?

Reply:
In the `https://substrate-ui.parity.io/#/settings`, set  `remote node/endpoint to connect to` be local node.